src/sfctl/commands.py

```diff
@@ -256,6 +256,20 @@ def load_command_table(self, args):  # pylint: disable=too-many-statements
             group.command('get', 'get_property_info')
             group.command('delete', 'delete_property')
 
+        with CommandGroup(self, 'events', client_func_path,
+                          client_factory=client_create) as group:
+            group.command('cluster', 'get_cluster_event_list')
```

Comment:
I wanted to keep the events as a separate group because that's how we've it in the REST APIs. it's not shown under the regular entities (because it can be called on deleted entities). so do you mean here in this group i should define it like: sfctl events application list & sfctl events all-applications list ? or sfctl events list-application-events & sfctl events list-all-applications-events ? or something else ? I wasn't able to come up with great naming schema anyway :), so all suggetions are welcome.

Reply:
More like just cluster-list or list-cluster instead of just the world cluster :)